src/renovation_tracker/main.py

```diff
@@ -168,56 +190,87 @@ async def create_renovation(renovation: Renovation, db: Annotated[Session, Depen
     except Exception as e:
         db.rollback()
         raise HTTPException(
-            status_code=500,
-            detail=f"Error occurred while creating renovation"
+            status_code=500, detail=f"Error occurred while creating renovation"
         )
 
 
 # READ renovations for given listing id
 @api.get("/listings/{listing_id}/renovations", response_model=list[RenovationRead])
 async def get_renovation(listing_id: int, db: Annotated[Session, Depends(get_db)]):
-    listing = db.query(models.Listing).filter(models.Listing.listing_id == listing_id).first()
+    listing = (
+        db.query(models.Listing).filter(models.Listing.listing_id == listing_id).first()
+    )
     if listing is None:
-        raise HTTPException(status_code=404,detail=f"Listing with id {listing_id} not found")
+        raise HTTPException(
+            status_code=404, detail=f"Listing with id {listing_id} not found"
+        )
     return listing.renovations
 
 
 # READ renovations for given renovation id
 @api.get("/renovations/{renovation_id}", response_model=RenovationRead)
-async def get_renovationWID(renovation_id: int, db: Annotated[Session, Depends(get_db)]):
-    renovation = db.query(models.Renovations).filter(models.Renovations.renovation_id == renovation_id).first()
+async def get_renovationWID(
```

Comment:
Can be slightly difficult to read
```suggestion
async def get_renovation_by_id(
```